src/main/scala/devices/uart/UART.scala

```diff
@@ -126,6 +126,10 @@ abstract class UART(busWidthBytes: Int, val c: UARTParams, divisorInit: Int = 0)
     UARTCtrlRegs.div -> Seq(
       RegField(c.divisorBits, div,
                  RegFieldDesc("div","Baud rate divisor",reset=Some(divisorInit))))
+
+    UARTCtrlRegs.txdone -> Seq(
+      RegField(1, txm.io.done,
+                 RegFieldDesc("txdone","Transmit finished",reset=Some(1))))
```

Comment:
this is a read-only register, so you should use  `RegField.r(...)`. Also, should set `volatile=true` inside the RegFieldDesc because this may change between reads to it. (note that the RegField.r(...) will automatically apply the Read-Only attribute to the description).

Reply:
Thanks, I believe these should be fixed.